containers/fluentbit/extra.conf

```diff
@@ -12,4 +12,11 @@
     HTTP_Passwd     ${ES_PASS}
     Logstash_Format On
     Logstash_Prefix firelens
-    tls             On
\ No newline at end of file
+    tls             On
+
+    # By default, Fluentbit sends the document type "flb_type".  Elasticsearch
+    # is gradually deprecating multiple types per index, and the index template
```

Comment:
[As of V8](https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html), which we will hopefully be able to upgrade to nicely because of paying attention to this.
💯